pages/dmp_supporting_info/support_04_store_protect.md

```diff
@@ -125,29 +125,29 @@ Please consult the institutional storage guides for details:
 	* Some applications may have their own data structure that can only be accessed through the application. Examples: some Electronic Lab Notebook (ELN) or Electronic Data Capture (EDC) applications
 
 #### Backups
-If you are using institutional solutions, the institution commonly provides backup and snapshotting functionality, that allow you to restore data. Applicable information should be provided by your IT department. It is good to be aware of the backup routines and frequency, and possible limitations of these before the need to recover data appears in your project.
+If you use institutional solutions, the institution commonly provides backup and snapshotting functionality, that allow you to restore data. Applicable information should be provided by your IT department. It is good to be aware of the backup routines and frequency, and possible limitations of these before the need to recover data appears in your project.
 
-If you are using other solutions you should check the backup routines of theses solutions, or you may have to implement your own routines.
-If not relying on managed storage, an often suggested backup rule is 3-2-1.
+If you use other solutions you should check the backup routines of theses solutions, or you may have to implement your own routines.
+If you are not relying on managed storage, an often suggested backup rule is 3-2-1.
 
 Here the explanation from [NFDI4Chem](https://www.nfdi4chem.de/3-2-1-rule/) [![License: CC BY-SA 4.0](https://img.shields.io/badge/License-CC_BY--SA_4.0-lightgrey.svg)](https://creativecommons.org/licenses/by-sa/4.0/)
 > The 3-2-1 Backup Rule in Detail
 > * Three Copies of the Data: In research, this means that in addition to the original data, create at least two additional copies. This redundancy ensures that even in the event of hardware failure or data corruption, backup versions are always available for access.
 > * Two Different Storage Media: It is advisable to back up the data on two different storage media. This could be, for example, a local server and an external hard drive. Using different media minimizes the risk that both copies could be lost simultaneously due to the same event (such as hardware failure). Do not rely on hard drives only!
-> * One Copy at an External Location: Store this third copy at a different physical location to protect it from local disasters such as fires or floods. While cloud storage solutions are often chosen, we recommend central storage, for example, at central IT or library services of the university/institution. Because they take care of the server housing and hosting, and the backups.
+> * One Copy at an External Location: Store this third copy at a different physical location to protect it from local disasters such as fires or floods. While cloud storage solutions are often chosen, we recommend central storage, for example, at central IT or library services of the university/institution, as they take care of the server housing, hosting and backups.
```

Comment:
One Copy at an External Location: Store this third copy at a different physical location to protect it from local disasters such as fires or floods. While cloud storage solutions are often chosen, we recommend central storage, for example, at central IT or library services of the university/institution. Because they take care of the server housing and hosting, and the backups.